internal/policies/mount/mount.go

```diff
@@ -464,9 +465,9 @@ func chown(p string, f *os.File, uid, gid int) (err error) {
 
 // cleanup removes the files generated when applying the mount policy to an object.
 func (m *Manager) cleanup(ctx context.Context, objectName string, isComputer bool) (err error) {
-	defer decorate.OnError(&err, gotext.Get("failed to clean up mount policy files for %q", objectName))
+	defer decorate.OnError(&err, gotext.Get("failed to clean up mount policy files for %q", objectName)) //nolint:govet // printf: this is an i18n formatted const string
 
-	log.Debugf(ctx, gotext.Get("Cleaning up mount policy files for %q", objectName))
+	log.Debugf(ctx, gotext.Get("Cleaning up mount policy files for %q", objectName)) //nolint:govet // printf: this is an i18n formatted const string
```

Comment:
Shouldn't we standardize how we use `//nolint` directives? It feels a bit off to have some of them at EOL and some above.

Reply:
you are completely correct. Thhis was a leftover, I will fix it.